src/delivery/forms.py

```diff
@@ -15,9 +15,24 @@ class DishIngredientsForm(forms.Form):
     )
 
     ingredients = forms.ModelMultipleChoiceField(
-        label=_('Select Ingredients:'),
+        label=_('Select Main Dish Ingredients:'),
         queryset=Ingredient.objects.order_by(Lower('name')).all(),
         widget=forms.SelectMultiple(
             attrs={'class': 'ui fluid search dropdown'}),
         required=False,
     )
+
+    sides_ingredients = forms.ModelMultipleChoiceField(
+        label=_('Select Sides Ingredients:'),
```

Comment:
No need to use camel case here. Just: _Select sides ingredients_ should be fine.